src/App.js

```diff
@@ -77,12 +78,43 @@ class App extends React.Component {
 			};
 		});
 	}
-	// addColorCircles([key]){
-	// 	this.setState 
-	// 	let circleClass = '';
-	// 	return ([key]) ? circleClass='card__item-list' : circleClass='card__item-list-no-border'
-	// };
+
+	sendRequest(buttonShare){
+		buttonShare.disabled = true;
+		console.log(this.state.profile)
+    fetch('https://us-central1-awesome-cards-cf6f0.cloudfunctions.net/card/', {
+			method: 'POST',
+			body: JSON.stringify(this.state.profile),
+			headers: {
+				'content-type': 'application/json'
+			},
+    })
+		.then(function (resp) { buttonShare.disabled = false; return resp.json(); })
+		.then(function (result) { console.log(result); })
+		.catch(function (error) { console.log(error); });
+	}
+	handlerButtonShare(event){
+		const buttonShare = event.currentTarget;
```

Comment:
Maybe you should add some comments to understand this logic of the button, or if not, explain it in the PR or in the Trello, or somewhere... :)